sdk/src/azure/core/az_http_policy.c

```diff
@@ -31,7 +31,7 @@ AZ_NODISCARD az_result az_http_pipeline_policy_apiversion(
     case _az_http_policy_apiversion_option_location_queryparameter:
       // Add the version as a query parameter
       AZ_RETURN_IF_FAILED(az_http_request_set_query_parameter(
-          ref_request, options->_internal.name, options->_internal.version));
+          ref_request, options->_internal.name, options->_internal.version, true));
```

Comment:
Why are we certain that we should pass `true` here? Add a comment.

Reply:
because the version doesn't need to be url-encoded.  We are saying that it is already url-encoded.
I can add the comment

Reply:
Yes please. Any place where you are opting out of encoding, should be explicitly commented.

Places where you are encoding, can be left as is (since I consider that as the default).